### PR TITLE
Add MassFlow.PoundPerHour

### PR DESCRIPTION
**Pull request by Faizal-Majeed**

### CODE REVIEW

File: UnitsNet.Tests/CustomCode/MassFlowTests.cs [hunk 1]
```diff
@@ -48,6 +48,8 @@ public class MassFlowTests : MassFlowTestsBase
 
         protected override double TonnesPerDayInOneGramPerSecond => 60.0*60*24/1E6;
 
+        protected override double PoundsPerHourInOneGramPerSecond => 0.125998;
```
Comment (anjdreas) [inline UnitsNet.Tests/CustomCode/MassFlowTests.cs]: Google says: `7.93664`
https://www.google.no/search?q=pounds+Per+Hour+In+1+Gram+Per+Second

File: UnitsNet/UnitDefinitions/MassFlow.json [hunk 2]
```diff
@@ -19,8 +19,8 @@
     {
       "SingularName": "KilogramPerHour",
       "PluralName": "KilogramsPerHour",
-      "FromUnitToBaseFunc": "x/3.6",
-      "FromBaseToUnitFunc": "x*3.6",
+      "FromUnitToBaseFunc": "x/0.277778",
```
Comment (anjdreas) [inline UnitsNet/UnitDefinitions/MassFlow.json]: I don't think this change is right. Google says `3.6`.

https://www.google.no/search?q=kilograms+Per+Hour+In+1+Gram+Per+Second

File: UnitsNet/UnitDefinitions/MassFlow.json [hunk 3]
```diff
@@ -32,6 +32,18 @@
         }
       ]
     },
+	{
+      "SingularName": "PoundPerHour",
+      "PluralName": "PoundsPerHour",
+      "FromUnitToBaseFunc": "x/0.125998",
+      "FromBaseToUnitFunc": "x*0.125998",
```
Comment (anjdreas) [inline UnitsNet/UnitDefinitions/MassFlow.json]: As commented in test, I think you want to switch these two around.

image

**Comment by anjdreas** [comment]
Thanks, looks good!

**Comment by Faizal-Majeed** [comment]
Thanks

**Comment by anjdreas** [comment]
Nuget 3.55 out

**Pull request merged**